src/actions/importFiles.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
This is fine. `deleteResumableFIle` is a somewhat hacky solution and should probably be refactored separately.